src/SFA.DAS.EmployerFinance.UnitTests/Events/ProcessDeclarationTests/WhenIProcessDeclarations.cs

```diff
@@ -49,5 +56,39 @@ public async Task ThenTheLoggerIsCalledWithInfoLevel()
             //Assert
             _logger.Verify(x=>x.Info("Process Declarations Called"));
         }
+
+        [TestCase(100023,0.0,false)]
+        [TestCase(900034,10.0,true)]
+        public async Task ThenLevyAddedToAccountEventIsPublishedForGreaterThanZeroLevyAmount(
```

Comment:
I would split this into 2 tests (positive and negative) as it's confusing otherwise.

Reply:
Split into three (levyAmount greater then, less than and equal to zero) now.

Oh and this is just straight equality `levyAmount == decimal.zero` do we need to consider tolerance equality?

I haven't seen examples of tolerance equality in the code, but thought it worth asking.